src/Model/CustomerProxy.php

```diff
@@ -2,10 +2,51 @@
 
 namespace Mapado\Stripe\Model;
 
+use Mapado\Stripe\Model\ChargeProxy as ChargeProxy;
+use Mapado\Stripe\Model\InvoiceProxy as InvoiceProxy;
+
 class CustomerProxy extends StripeObject
 {
     public function __get($key)
     {
         return $this->stripeObject->{$key};
     }
+
+    /**
+     * getChargeList
+     *
+     * @param array $params
+     * @access public
+     * @return array
+     */
+    public function getChargeList(array $params = array())
```

Comment:
L'inverse serait peut être plus logique, laisser la méthode dans le customer et l'appeler depuis l'API ?
Le 3 mars 2015 19:26, Julien Deniau <notifications@github.com> a écrit :In src/Model/CustomerProxy.php:
>  class CustomerProxy extends StripeObject
>  {
>      public function __get($key)
>      {
>          return $this->stripeObject->{$key};
>      }
> +
> +    /**
> +     * getChargeList
> +     *
> +     * @param array $params
> +     * @access public
> +     * @return array
> +     */
> +    public function getChargeList(array $params = array())

en fait tu peux simplement faire un $this->api->getChargeListForUser($this, $params) Idem pour l'autre méthode

—Reply to this email directly or view it on GitHub.